File: JAVA/kibeompark/java_work/src/Homework/Seventh/Homework1.java
package Homework.Seventh;

class OnetoTen {
    public void Print() {
        for (int i = 1; i <= 10; i++) {
            System.out.println(i);
        }
    }
}

public class Homework1 {
    public static void main(String[] args) {
        // 1) 1 ~ 10 까지 출력하는 클래스 내의 메서드를 만들어 보자 !
        OnetoTen ot = new OnetoTen();

        ot.Print();
    }
}

File: JAVASCRIPT/junghyunlee/js_test/src/homework/Homework1.js
const Homework1 = () => {
    const START = 1;
    const END = 20;

    for(var i = START; i <= END; i++) {
        console.info("Homework: " + i)
    }

    return (
        <div className="Homework1">
            <p>
                Homework 1
            </p>
        </div>
    )
}

export default Homework1